package.json

```diff
@@ -12,7 +12,7 @@
     "profile": "node --prof ./bin/server",
     "functional": "cucumber-js --format=progress --tags '(not @ignore) and (not @beta)' --retry 2 --fail-fast --exit",
     "functional-beta": "cucumber-js --format=progress --tags '(not @ignore) and (not @prod)' --retry 2 --fail-fast --exit",
-    "functional-dev": "cucumber-js --format=./node_modules/cucumber-pretty --tags '(not @ignore) and (not @prod)' --retry 2 --fail-fast --exit",
+    "functional-dev": "cucumber-js --format=progress --tags '(not @ignore) and (not @prod)' --retry 2 --fail-fast --exit",
```

Comment:
`cucumber-pretty` was used to ignore the Remote Join test with [this PR](https://github.com/screwdriver-cd/screwdriver/pull/2171), but ignore has already been removed with [this PR](https://github.com/screwdriver-cd/screwdriver/pull/2193).